scripts/stack.ts

```diff
@@ -13,8 +13,8 @@ import {
   Aspects,
 } from "@aws-cdk/core"
 import { name } from "../package.json"
-import { ENV } from "../src/util"
```

Comment:
Removed the reference to lambda runtime env variables and updated logic in file to look up env variables at stack create run time before deploy